services/storage-service.ts

```diff
@@ -188,19 +200,20 @@ export const getUserId = async () => {
  * @returns {Promise<Array>} A promise that resolves with a list of courses.
  */
 export const getCourseList = async () => {
-  let courseList = [];
+  let courseList: any[] = [];
```

Comment:
```suggestion
  let courseList: Course[] = [];
```

...with global `Course` type in `types/course.ts`